### ISSUE TITLE
Failed to load the sync-settings package

### ISSUE THREAD

**Issue by cassianoleal** [Hackafe/atom-sync-settings#211]
Error happens right upon opening Atom.

**Atom Version**: 1.5.4
**System**: Mac OS X 10.11.3
**Thrown From**: [sync-settings](https://github.com/Hackafe/atom-sync-settings) package, v0.7.0

### Stack Trace

Failed to load the sync-settings package

```
At Cannot find module 'underscore-plus'

Error: Cannot find module 'underscore-plus'
    at Module._resolveFilename (module.js:336:15)
    at Function.Module._resolveFilename (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/Users/cassiano/.atom/packages/sync-settings/lib/sync-settings.coffee:4:5)
    at Object.<anonymous> (/Users/cassiano/.atom/packages/sync-settings/lib/sync-settings.coffee:2:1)
    at Module._compile (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/package.js:720:27)
    at /opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/package.js:116:28
    at Package.module.exports.Package.measure (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at Package.module.exports.Package.load (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/package.js:106:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/package-manager.js:445:14)
    at PackageManager.module.exports.PackageManager.loadPackages (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/package-manager.js:395:14)
    at AtomEnvironment.module.exports.AtomEnvironment.startEditorWindow (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:684:21)
    at module.exports (/opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/src/initialize-application-window.js:28:10)
    at setupWindow (file:///opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/static/index.js:86:5)
    at window.onload (file:///opt/homebrew-cask/Caskroom/atom/1.4.3/Atom.app/Contents/Resources/app.asar/static/index.js:41:9)
```

### Commands

```
     -0:06.5.0 core:backspace (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "vertical-align",
      "term2",
      "autocomplete-ruby",
      "welcome",
      "metrics",
      "language-toml",
      "language-sass",
      "language-perl",
      "language-objective-c",
      "language-less",
      "language-javascript",
      "language-java",
      "language-git",
      "language-csharp",
      "language-coffee-script",
      "language-clojure",
      "language-c",
      "keybinding-resolver",
      "language-mustache",
      "valign",
      "minimap-autohide"
    ],
    "themes": [
      "one-outlander-ui",
      "material-design-syntax"
    ]
  },
  "sync-settings": {
    "_analyticsUserId": "3e117e10-c24a-48bf-8cd3-791c5794c186",
    "_lastBackupHash": "95c5071d4d6139d54476cbe9a67ffcdcf2c30cfe",
    "gistId": "3d4f61324de066483de4",
    "personalAccessToken": "c2c431fcb69a4cfdf8600657d47ddd72887e6b42"
  }
}
```

### Installed Packages

```coffee
# User
aligner, v0.19.1
aligner-puppet, v0.1.1
aligner-python, v1.0.1
aligner-ruby, v1.4.0
atom-cli-diff, v1.3.0
atom-erb, v0.2.0
atom-pair, v2.0.10
atom-python-debugger, v0.3.4
auto-indent, v0.5.0
autocomplete-paths, v1.0.2
autocomplete-python, v1.7.0
docs-snippets, v0.8.0
editorconfig, v1.2.4
file-watcher, v0.4.0
git-log, v0.4.1
git-projects, vundefined
git-tab-status, v1.9.2
highlight-selected, v0.11.2
language-apache, v1.4.0
language-bats, v0.1.0
language-chef, v0.8.0
language-docker, v1.1.6
language-puppet, v0.17.0
language-rspec, v0.5.0
language-scala, v1.1.6
language-terraform, v0.7.2
language-x86-64-assembly, v2.1.5
line-ending-converter, v1.3.2
linter, v1.11.3
linter-erb, v1.0.2
linter-foodcritic, v0.5.2
linter-js-yaml, v1.2.5
linter-jshint, v2.0.2
linter-puppet-lint, v0.6.2
linter-puppet-parse, v0.6.0
linter-ruby, v1.2.2
linter-shellcheck, v1.1.0
material-design-syntax, v1.7.0
minimap, v4.20.0
minimap-bookmarks, v0.3.0
minimap-color-highlight, v4.1.4
minimap-find-and-replace, v4.5.0
minimap-git-diff, v4.2.0
minimap-highlight-selected, v4.4.0
minimap-linter, v1.1.1
minimap-selection, v4.4.0
motepair, v0.21.0
one-outlander-ui, v0.3.1
project-colorize, v0.2.0
project-manager, v2.7.6
project-view, v0.5.0
python-iresolve, v0.0.3
python-tools, v0.6.7
remote-edit, v1.8.24
rest-client, v0.5.1
rspec-snippets, v0.4.0
ruby-block, v0.3.5
serverspec-snippets, v0.1.0
ssh-config, v0.10.1
sync-settings, v0.7.0
synced-sidebar, v0.4.3
trailing-spaces, v0.3.2

# Dev
No dev packages
```

**Comment by jerone** [comment]
Duplicate of https://github.com/Hackafe/atom-sync-settings/issues/194.

In the above error report you have shared your `personalAccessToken`. You must revoke it to prevent abuse.

**Comment by cassianoleal** [comment]
Ugh! Thanks for pointing that out. Revoked. :smile:

**Comment by xenithorb** [comment]
Restarting does not alleviate this issue for me on linux, worse, the "View issue" button always brings me back to this issue.

**Comment by dirk-thomas** [comment]
Please provide more information about your problem: the Atom and sync-settings version you are using, the platform you are on, what settings you have configured, what the exact error is, and any output from the debug console might be helpful.

**Comment by xenithorb** [comment]
Unfortunately I already worked around the issue, and can't replicate it anymore. The fix was to use `apm uninstall sync-settings && apm install sync-settings` with atom closed, I tried this simply because someone mentioned elsewhere that this could be due to locked files.. that at least worked to get the missing module installed.

I'm on Fedora 25 using atom `1.12.7` and sync-settings `0.8.0`

The issue appeared after returning to my workstation after some days of away-time and doing 5-6 package updates simultaneously. I'm not sure which, though, but I wanted to include that it was several.
